cli/cmd/integration.go

```diff
@@ -71,11 +72,30 @@ var (
 
 	// integrationCreateCmd represents the create sub-command inside the integration command
 	integrationCreateCmd = &cobra.Command{
-		Use:    "create",
-		Hidden: true,
-		Short:  "create an external integrations",
-		Args:   cobra.NoArgs,
+		Use:   "create",
+		Short: "create an external integrations",
+		Args:  cobra.NoArgs,
+		Long: `This command will prompt an interactive session that will help you create
```

Comment:
I think rather than "This command will..." we should just be explicit...

`Creates an external integration in your account through an interactive session. Use the '--noninteractive' flag to pass disable interactive mode`

@afiunelw I also see that `lacework int create --help` does not specify which parameters you need to pass if you use `--noninteractive`

Reply:
I will change the message, it is trying to let the user know that, the command is disabled if the CLI is running in noninteractive mode.